Readme.md

```diff
@@ -113,5 +129,7 @@ docker-compose volume rm <volume name>
 
 ## Todo
```

Comment:
Can we link to the github issues with the feature tag, otherwise we're maintaining this list in two places.

Reply:
Yeah, I thought the same thing! I will just add a note in this section instead of the list. And add the items here as issues if they aren't already there.